src/basic/REDUCE_STRUCT.cpp

```diff
@@ -0,0 +1,91 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-21, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "REDUCE_STRUCT.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include "common/DataUtils.hpp"
+
+#include <limits>
+
+namespace rajaperf
+{
+namespace basic
+{
+
+
+REDUCE_STRUCT::REDUCE_STRUCT(const RunParams& params)
+  : KernelBase(rajaperf::Basic_REDUCE_STRUCT, params)
+{
+  setDefaultProblemSize(100);
+//setDefaultReps(5000);
+// Set reps to low value until we resolve RAJA omp-target
+// reduction performance issues
+  setDefaultReps(50);
```

Comment:
@rhornung67 Is this still necessary?

Reply:
It is for OpenMP target variants since our reductions for that case are not implemented properly. We should probably guard those with RAJA_ENABLE_TARGET_OPENMP so that only applies to that case.